velox/dwio/parquet/tests/ParquetTpchTest.cpp

```diff
@@ -123,6 +123,20 @@ class ParquetTpchTest : public testing::Test {
     ASSERT_EQ(expectedFinishedSplits, stats.numFinishedSplits);
   }
 
+  void assertQuery(
+      int queryId,
```

Comment:
Change this variable to veloxQueryId. It might be cleaner to make this an optional parameter with a default value as the last parameter of this function. That would avoid having 2 almost similar functions.